src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+function sortList(list) {
+  const employees = Array.from(list);
+
+  employees.sort((a, b) => {
+    const salaryA = parseFloat(a.dataset.salary.replace(/[$,]/g, ''));
+    const salaryB = parseFloat(b.dataset.salary.replace(/[$,]/g, ''));
+
+    return salaryB - salaryA;
+  });
+
+  const listContainer = document.querySelector('ul');
```

Comment:
Ensure that the `ul` element exists in the DOM before attempting to append children to it. Consider adding a check to handle cases where `listContainer` might be `null`.